python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -0,0 +1,246 @@
+#
+# LSST Data Management System
+# Copyright 2019 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+import lsst.afw.image as afw_image
+
+from ..locateImage import get_image
+from .soda.soda import SODA
+
+""" This module implements the IVOA's SODA v1.0 for DAX ImageServ.
+
+    All ra,dec values are expressed in ICRS degrees, by default.
+
+    Shape:
+        CIRCLE <ra> <dec> <radius>
+        RANGE <ra1> <ra2> <dec1> <dec2>
+        POLYGON <ra1> <dec1> ... (at least 3 pairs)
+        BRECT <ra> <dec> <w> <h> <filter> <unit>
+"""
+
+
+class ImageSODA(SODA):
+    """ Class to handle SODA operations.
+
+    For reference, LSST camera filters (Filter, Blue Side, Red Side):
+        G : 400 - 552
+        R : 552 - 691
+        I : 691 - 818
+        Z : 818 - 922
+        Y : 948 - 1060
+    """
+    def __init__(self, config, dashboard):
+        self._config = config
+        self._dashboard = dashboard
+
+    def do_sync(self, params: dict) -> object:
+        """ Do sync operation.
+
+        Parameters
+        ---------
+        params : `dict`
+            the HTTP parameters
+        """
+        if "POS" in params:
+            resp = super().handle_pos(params)
+            return resp
+        else:
+            return "Unsupported Request"
+
+    def do_async(self, params: dict) -> object:
+        """ Do async operation.
+
+        Parameters
+        ----------
+        params : `dict`
+            the HTTP parameters.
+
+        Returns
+        -------
+        xml: `str`
+
+        """
+        resp = "OK"
```

Comment:
Does this mean we don't support ASYNC?  We just return "OK?"  Maybe throw not implemented or something?